h2/events.py

```diff
@@ -146,6 +146,17 @@ class _HeadersSent(object):
     pass
 
 
+class _ResponseHeadersSent(_HeadersSent):
+    """
+    The _ResponseHeadersSent event is fired whenever response headers are sent
+    on a stream.
+
+    This is an internal event, used to determine validation steps on
+    outgoing header blocks.
+    """
+    pass
```

Comment:
I'd like at this point to also add a `_RequestHeadersSent` field. We don't have to do it as part of this PR, but can you open a new issue to track adding the new internal event? The lack of symmetry is making me sad. ;)